File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.java
class Solution {
    public int solution(int a, int b, int n) {
        int answer = 0;

        while (n >= a) {
            int tmp1 = n / a * b;
            n = tmp1 + n % a;
            answer += tmp1;
        }

        return answer;
    }
}

File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.py
def solution(a, b, n):
    answer = 0
    while n >= a:
        tmp = n // a * b # 빈병 구하기
        n = tmp + n % a # 빈병이랑 빈병이 a보다 작을때 나머지 값
        answer += tmp # 빈병 더하기
    return answer